packages/pigeon/lib/ast.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:collection/collection.dart' show ListEquality;
 import 'package:meta/meta.dart';
+import 'package:pigeon/pigeon_lib.dart';
```

Comment:
Shouldn't this just be `import 'pigeon_lib.dart';`?

Reply:
done